src/Plaid/Management/PlaidLinkEventName.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Going.Plaid.Management
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	/// <remarks>See: <see href="https://plaid.com/docs/#event-names"/></remarks>
+	public enum PlaidLinkEventName
+	{
+		/// <summary>
+		/// The user closed the third-party website or mobile app without completing the OAuth flow.
+		/// </summary>
+		CLOSE_OAUTH,
```

Comment:
Would prefer to see `CloseOAuth` with a `[EnumMember]` attribute. See `InvestmentTransactionType` for example.

Reply:
I can change that, but doesn't that just add complexity and make reading the raw data and resulting deserialized values 1 extra step more complicated? I.e. if you view the raw values you cannot just look for that value in the Enum, you have to look at the source code for the EnumMember and if someone is serializing or deserializing the enum they have to be sure the serialization code can use the EnumMember attribute. I.e. one more thing that can break.

Reply:
Two different comments:
1. The goal of the library is to abstract the raw to make it easier to use in C#. This includes rewriting enum values from whatever base format is used externally into the C# CamelCase formatting. If done properly, users should not be overly concerned w/ the raw values. Instead, they'll receive a value in the C# object as they debug and can translate to what they need.
2. Plaid has made a commitment that many of these ENUM values will not change. Display messages may change, but types and codes will not. Changing the code would be considered a breaking change.

Take a look at the webhook-entity branch and the error-reporting branch. I'm doing the webhook-entity one for #8, and when I saw what was missing on the error side, I started working on that as well.

Reply:
I personally think it's more trouble debugging when searching through Plaid docs for the error code. I can't just copy and paste from my serialized log entries, I have to find the right source code and then look for the EnumMember and THEN search for that value in Plaid docs ... and it REQUIRES me to have the library source code when debugging vs a user of the library who is not intending to debug deeply into Plaid, just wants to use the library...

But I realize that's a subjective opinion and I'm not going to to the wall for it.

However, on the [https://github.com/viceroypenguin/Going.Plaid.NET/tree/PlaidLink-Metadata](PlaidLink-Metadata) branch there are a other Enums for which you will have the same feedback. I'm not sure if/when I'll have time to go reformat them all (one is really long). They were pretty useful for me for logging in CloudWatch as I can do JSON searches there for serialized data structures.

(if I'm bored some day I will try and build the RegEx to apply to the source files to reformat them as CamelCase w EnumMember... but I'm busier for the next couple of months than the past few)